packages/node/src/indexer/fetch.service.ts

```diff
@@ -643,26 +644,34 @@ export class FetchService implements OnApplicationShutdown {
   }
 
   private dictionaryValidation(
-    { _metadata: metaData }: Dictionary,
-    startBlockHeight: number,
+    dictionary: Dictionary | SpecVersionDictionary,
```

Comment:
We can use the minimal data type here, means its easier to determine what data is used and easier to write tests
```suggestion
    { _metadata}: { _metadata: MetaData },
```